F/projects section

JohnRamberger opened the pull request:
Added a navbar item for projects
![image](https://user-images.githubusercontent.com/56486927/144336386-3e306ff5-9606-412c-99fb-91e9dffb1839.png)

Added a projects section to the site (with placeholder image for now)
![image](https://user-images.githubusercontent.com/56486927/144336476-b68c118f-0b98-4d03-bb27-cdc765ea674f.png)

rishsol commented:
Can you do the opposite orientation for the corner blocks, the first section begins with top l, bottom r and then it alternates
also, it would be nice to have it say in development somewhere on the project block

rishsol reviewed (changes requested):
Can you do the opposite orientation for the corner blocks, the first section begins with top l, bottom r and then it alternates
also, it would be nice to have it say in development somewhere on the project block

JohnRamberger commented:
Changed the corner block orientation to the opposite, and added 'in development' on the projects.

![image](https://user-images.githubusercontent.com/56486927/144339990-aa8b1d6c-094e-4da1-b47d-8b387914c6dc.png)
